app/Model/Topic.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Model;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Topic extends Model
+{
+    protected $fillable = [
+        "name",
+    ];
+    public function lesson()
```

Comment:
hasMany mình đặt tên funtion số nhiều e nhé
1 Topic có nhiều lessons
```suggestion
    public function lessons()
```